app/services/trust_update_service.rb

```diff
@@ -4,18 +4,42 @@ def update_trusts(last_update: nil)
     last_update ||= DataStage::DataUpdateRecord.last_update_for(:trusts)
 
     # simple updates for trusts that are open
+    update_open_trusts(last_update)
+
+    # auto close trusts that have no schools
+    # skip and notify sentry when a trust is closed and still has schools - needs investigation
+    close_trusts(last_update)
+
+    DataStage::DataUpdateRecord.updated!(:trusts)
+  end
+
+private
+
+  def update_open_trusts(last_update)
     DataStage::Trust.updated_since(last_update).gias_status_open.each do |staged_trust|
       trust = Trust.find_by(companies_house_number: staged_trust.companies_house_number)
 
       next unless trust
```

Comment:
Whilst here, is it worth logging when this happens? I know you mentioned posting a notification e.g. to Sentry (though I may have misunderstood about which case to do so), but a simple `warn` log entry could allow us to grep for when this occurs?

Reply:
Yeah I think more general logging is needed here so we can see what happened each day.